### PR TITLE
Restored missing tree templates

### PR DESCRIPTION
**Pull request by thierrymarianne**
| Questions | Answers |
| --- | --- |
| Branch? | develop |
| Description? | When adding a new category, browsing the category tree should not raise a templating error |
| Type? | bug fix |
| Category? | BO |
| BC breaks? | no |
| Deprecations? | no |
| Fixed ticket? | http://forge.prestashop.com/browse/BOOM-1145 |
| How to test? | Expand the category tree when adding a new category |

### CODE REVIEW

File: admin-dev/themes/default/template/helpers/tree/subtree_associated_categories.tpl [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+* to license@prestashop.com so we can send you a copy immediately.
+*
+* DISCLAIMER
+*
+* Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+* versions in the future. If you wish to customize PrestaShop for your
+* needs please refer to http://www.prestashop.com for more information.
+*
+* @author    PrestaShop SA <contact@prestashop.com>
+* @copyright 2007-2015 PrestaShop SA
```
Comment (mickaelandrieu) [inline admin-dev/themes/default/template/helpers/tree/subtree_associated_categories.tpl]:
```diff
- @copyright 2007-2015 PrestaShop SA
+ @copyright 2007-2016 PrestaShop SA
```

File: admin-dev/themes/default/template/helpers/tree/subtree_associated_categories.tpl [hunk 2]
```diff
@@ -0,0 +1,55 @@
+{*
+* 2007-2015 PrestaShop
```
Comment (mickaelandrieu) [inline admin-dev/themes/default/template/helpers/tree/subtree_associated_categories.tpl]:
```diff
- * 2007-2015 PrestaShop
+ * 2007-2016 PrestaShop
```

**Comment by vincentbz** [comment]
Tested and ok, thanks !

**Comment by mickaelandrieu** [comment]
Thank you @thierrymarianne !

**Pull request merged**